cpp/src/io/csv/csv_gpu.cu

```diff
@@ -224,7 +224,7 @@ __global__ void __launch_bounds__(csvparse_block_dim)
       long tempPos   = pos - 1;
       long field_len = pos - start;
 
-      if (field_len <= 0 || serialized_trie_contains(opts.trie_na, raw_csv + start, field_len)) {
+      if (field_len < 0 || serialized_trie_contains(opts.trie_na, raw_csv + start, field_len)) {
```

Comment:
I guess `field_len < 0` is because we're using inclusive offsets (i.e, [0, 0] includes index 0)... Do we have a bug open for changing inclusive offsets -> exclusive offsets throughout cuio?

Reply:
Might want to add a comment to this so it's not thought to be a mistake. Or use `<= -1` ?

Reply:
Took another look and it seems like `field_len` can never be negative. Since `long field_len = pos - start;`, and pos is initialized to start and can only advance.
I left the condition as a sanity check, but it actually makes no sense in the context. Will remove the condition, thanks!